pkg/driver/camera/camera_darwin.go

```diff
@@ -0,0 +1,71 @@
+package camera
```

Comment:
all these MacOS specific code requires `// +build darwin`

Reply:
I think that should be automatic, https://golang.org/pkg/go/build/#hdr-Build_Constraints. Since I named the file with `_darwin` prefix, it should be only for darwin?

Reply:
Oops, yes, this file is fine.
But, `pkg/avfoundation/avfoundation_callback.go` needs that to avoid failing on `go build ./...`.

Reply:
Oops...

Reply:
@at-wat updated. PTAL